src/ModelContextProtocol/ModelContextProtocol.csproj

```diff
@@ -31,6 +31,11 @@
     <PackageReference Include="System.IO.Pipelines" />
   </ItemGroup>
 
+  <!-- Dependencies only needed by netstandard2.0 or net8.0 -->
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0' or '$(TargetFramework)' == 'net9.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Http"/>
```

Comment:
We're definitely not interested in adding any Microsoft.AspNetCore to this project. That's why ModelContextProtocol.AspNetCore exists. If you want to access the HttpContext in a tool call, I recommend using IHttpContextAccessor, at least for now, as demonstrated in in MapMcpTests.cs.

https://github.com/modelcontextprotocol/csharp-sdk/blob/9505eb516074364ff4fba02c3f4b760636697c8e/tests/ModelContextProtocol.AspNetCore.Tests/MapMcpTests.cs#L141-L151